README.md

```diff
@@ -8,13 +8,122 @@ Simply create a configuration file to define multiple services and databases to
 
 #### Currently supports
 
-## Monitoring
+- HTTP - Headers and all that
+- Postgresql
+- MySQL
+- MariaDB
+- MSSQL
+- Redis
 
+## Deployment
+
+
+Deployment options:
+- [Dockerfile](https://github.com/wavezync/pulse-bridge/blob/main/Dockerfile) – Self-hosted container registry or Docker Hub image
+- Kubernetes
+
+### 💻 Deploy locally (Build from source)
+
+```bash
+git clone https://github.com/wavezync/pulse-bridge.git
+cd pulse-bridge
+go run .
+```
+
+### 🐳  Deploy with Docker
+
+```bash
+git clone https://github.com/wavezync/pulse-bridge.git
```

Comment:
lets add our docker image here, so pople know how to use the image

also we need to provide binaries for deploying without docker too